sphinx/docserver.py

```diff
@@ -91,7 +91,7 @@ def serve_http():
     print(f"Visit http://localhost:{PORT}/en/latest/index.html to see plots\n")
 
     server = threading.Thread(target=serve_http)
-    server.start()
+    server.install()
```

Comment:
Here is another one that must be reverted. This is a Python `threading.Thread` not a Bokeh `Server`